docs/configuration.stories.mdx

```diff
@@ -94,78 +73,10 @@ export default Vue.extend({
 <style lang="scss">
 $color-primary: #f99;
 $color-secondary: #f55;
-@import '~bevitch/src/styles/lib.scss';
+@import '~@beVitch/core/src/styles/lib.scss';
 </style>
 ```
 
-Default settings:
```

Comment:
Do You have any idea where we are going to keep this info for users?

Reply:
Maybe it is a good place but i was not sure that these docs were actual and i think we must plan how to make documentation for all variables

Reply:
Yes they were. You are working with me :)
But i agree, we should rethink that.